{"name": "Test Case 2: Login con usuario existente", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x63441d96b2ca <unknown>\n#1 0x63441d412550 <unknown>\n#2 0x63441d4640f0 <unknown>\n#3 0x63441d4642e1 <unknown>\n#4 0x63441d4b25e4 <unknown>\n#5 0x63441d489bed <unknown>\n#6 0x63441d4af9e6 <unknown>\n#7 0x63441d489993 <unknown>\n#8 0x63441d455d6b <unknown>\n#9 0x63441d457141 <unknown>\n#10 0x63441d9302ab <unknown>\n#11 0x63441d9340b9 <unknown>\n#12 0x63441d917139 <unknown>\n#13 0x63441d934c68 <unknown>\n#14 0x63441d8fb60f <unknown>\n#15 0x63441d9591f8 <unknown>\n#16 0x63441d9593d6 <unknown>\n#17 0x63441d96a5e6 <unknown>\n#18 0x73a3c349caa4 <unknown>\n#19 0x73a3c3529c3c <unknown>", "trace": "browser = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"2c3ccf41c3e4ddfa4c6b0516970fdfde\")>\n\n    @allure.title(\"Test Case 2: Login con usuario existente\")\n    @allure.description(\"Este test verifica que un usuario puede loguearse correctamente y luego eliminar la cuenta.\")\n    def test_login_user_with_correct_credentials(browser):\n        home_page = HomePage(browser)\n        login_page = LoginPage(browser)\n        account_info_page = AccountInfoPage(browser)\n    \n        email = \"testeando@gmail.com\"\n        password = \"test\"\n        name = \"test\"  # Solo para validación del mensaje 'Logged in as'\n    \n        with allure.step(\"Navegar a la página principal\"):\n            home_page.go_to()\n            assert home_page.is_logo_visible()\n    \n        with allure.step(\"Hacer click en 'Signup / Login'\"):\n            home_page.click_signup_login()\n    \n        with allure.step(\"Verificar que aparece 'Login to your account'\"):\n            assert login_page.is_login_to_your_account_visible()\n    \n        with allure.step(\"Ingresar email y contraseña válidos\"):\n            login_page.enter_login_email(email)\n            login_page.enter_login_password(password)\n    \n        with allure.step(\"Hacer click en 'Login'\"):\n            login_page.click_login_button()\n    \n        with allure.step(\"Verificar que se muestra 'Logged in as <nombre>' en la Home\"):\n>           assert home_page.is_logged_in_as_visible(name), f\"No se encontró el mensaje 'Logged in as {name}'\"\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\ntests/test_auth.py:132: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npages/home_page.py:34: in is_logged_in_as_visible\n    element = self.wait_for_element(self.LOGGED_IN_AS_TEXT)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nbase/base_page.py:15: in wait_for_element\n    return WebDriverWait(self.driver, timeout).until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"2c3ccf41c3e4ddfa4c6b0516970fdfde\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x7cc265ec7d80>, message = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Wait until the method returns a value that is not False.\n    \n        Calls the method provided with the driver as an argument until the\n        return value does not evaluate to ``False``.\n    \n        Parameters:\n        -----------\n        method: callable(WebDriver)\n            - A callable object that takes a WebDriver instance as an argument.\n    \n        message: str\n            - Optional message for :exc:`TimeoutException`\n    \n        Return:\n        -------\n        object: T\n            - The result of the last call to `method`\n    \n        Raises:\n        -------\n        TimeoutException\n            - If 'method' does not return a truthy value within the WebDriverWait\n            object's timeout\n    \n        Example:\n        --------\n        >>> from selenium.webdriver.common.by import By\n        >>> from selenium.webdriver.support.ui import WebDriverWait\n        >>> from selenium.webdriver.support import expected_conditions as EC\n    \n        # Wait until an element is visible on the page\n        >>> wait = WebDriverWait(driver, 10)\n        >>> element = wait.until(EC.visibility_of_element_located((By.ID, \"exampleId\")))\n        >>> print(element.text)\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x63441d96b2ca <unknown>\nE       #1 0x63441d412550 <unknown>\nE       #2 0x63441d4640f0 <unknown>\nE       #3 0x63441d4642e1 <unknown>\nE       #4 0x63441d4b25e4 <unknown>\nE       #5 0x63441d489bed <unknown>\nE       #6 0x63441d4af9e6 <unknown>\nE       #7 0x63441d489993 <unknown>\nE       #8 0x63441d455d6b <unknown>\nE       #9 0x63441d457141 <unknown>\nE       #10 0x63441d9302ab <unknown>\nE       #11 0x63441d9340b9 <unknown>\nE       #12 0x63441d917139 <unknown>\nE       #13 0x63441d934c68 <unknown>\nE       #14 0x63441d8fb60f <unknown>\nE       #15 0x63441d9591f8 <unknown>\nE       #16 0x63441d9593d6 <unknown>\nE       #17 0x63441d96a5e6 <unknown>\nE       #18 0x73a3c349caa4 <unknown>\nE       #19 0x73a3c3529c3c <unknown>\n\ntestenv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:138: TimeoutException"}, "description": "Este test verifica que un usuario puede loguearse correctamente y luego eliminar la cuenta.", "steps": [{"name": "Navegar a la página principal", "status": "passed", "start": 1756735239152, "stop": 1756735241953}, {"name": "Hacer click en 'Signup / Login'", "status": "passed", "start": 1756735241953, "stop": 1756735243619}, {"name": "Verificar que aparece 'Login to your account'", "status": "passed", "start": 1756735243619, "stop": 1756735243649}, {"name": "Ingresar email y contraseña válidos", "status": "passed", "start": 1756735243649, "stop": 1756735243977}, {"name": "Hacer click en 'Login'", "status": "passed", "start": 1756735243977, "stop": 1756735244902}, {"name": "Verificar que se muestra 'Logged in as <nombre>' en la Home", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x63441d96b2ca <unknown>\n#1 0x63441d412550 <unknown>\n#2 0x63441d4640f0 <unknown>\n#3 0x63441d4642e1 <unknown>\n#4 0x63441d4b25e4 <unknown>\n#5 0x63441d489bed <unknown>\n#6 0x63441d4af9e6 <unknown>\n#7 0x63441d489993 <unknown>\n#8 0x63441d455d6b <unknown>\n#9 0x63441d457141 <unknown>\n#10 0x63441d9302ab <unknown>\n#11 0x63441d9340b9 <unknown>\n#12 0x63441d917139 <unknown>\n#13 0x63441d934c68 <unknown>\n#14 0x63441d8fb60f <unknown>\n#15 0x63441d9591f8 <unknown>\n#16 0x63441d9593d6 <unknown>\n#17 0x63441d96a5e6 <unknown>\n#18 0x73a3c349caa4 <unknown>\n#19 0x73a3c3529c3c <unknown>\n\n", "trace": "  File \"/home/gianfranco/Desktop/Proyecto/tests/test_auth.py\", line 132, in test_login_user_with_correct_credentials\n    assert home_page.is_logged_in_as_visible(name), f\"No se encontró el mensaje 'Logged in as {name}'\"\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/gianfranco/Desktop/Proyecto/pages/home_page.py\", line 34, in is_logged_in_as_visible\n    element = self.wait_for_element(self.LOGGED_IN_AS_TEXT)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/gianfranco/Desktop/Proyecto/base/base_page.py\", line 15, in wait_for_element\n    return WebDriverWait(self.driver, timeout).until(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/gianfranco/Desktop/Proyecto/testenv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1756735244902, "stop": 1756735255176}], "start": 1756735239151, "stop": 1756735255181, "uuid": "88e6deed-e31d-4155-ac3f-68c971145fb4", "historyId": "08ac928efea9217b50b7f3e383691e3f", "testCaseId": "08ac928efea9217b50b7f3e383691e3f", "fullName": "tests.test_auth#test_login_user_with_correct_credentials", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth"}, {"name": "host", "value": "gianfranco-h81h3m4"}, {"name": "thread", "value": "13164-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth"}], "titlePath": ["tests", "test_auth.py"]}